lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -7,26 +7,39 @@ define((require, exports, module) ->
 
   START_STATE = "start"
 
+  LIST_ITEMIZE_STATE = "list.itemize"
+  LIST_ITEMIZE_REGEX = "itemize"
+
+  LIST_ENUMERATE_STATE = "list.enumerate"
+  LIST_ENUMERATE_REGEX = "enumerate"
+
+  EQUATION_REGULAR_STATE = "equation.regular"
+  EQUATION_REGULAR_REGEX = "equation"
+
+  EQUATION_ASTERISK_STATE = "equation.asterisk"
+  EQUATION_ASTERISK_REGEX = "equation\\*"
+
+  MATH_MULTILINE_STATE = "math.multiline"
```

Comment:
maybe rename to `math.displayed.tex`, `math.inline.tex`, `math.displayed.latex` and `math.inline.latex` (that's `\(` and `\)` ) 
See therminology in wikibooks: https://en.wikibooks.org/wiki/LaTeX/Mathematics#Mathematics_environments